client-misk-dynamodb/README.md

```diff
@@ -0,0 +1,3 @@
+# client-misk-dynamodb
+
+Backfila Misk client implementation to backfill a DynamoDb datastore.
```

Comment:
supernit: DynamoDB is the spelling with an uppercase B. We use DynamoDb in code because its an acronym

Reply:
Yep done. Here and in comments it is DynamoDB. Where it is part of code it is DynamoDb or dynamoDb.